samtranslator/third_party/py27hash/hash.py

```diff
@@ -104,7 +106,7 @@ def fhash(value):  # type: ignore[no-untyped-def]
         """
 
         fpart = math.modf(value)
-        if fpart[0] == 0.0:
+        if fpart[0] == Hash._FLOAT_ZERO:
```

Comment:
Why is this prefixed by `Hash.`? Shouldn't `_FLOAT_ZERO` suffice?

Reply:
Its scope is still inside the class.